tools/e2e-commons/package.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "jetpack-e2e-commons",
+	"name": "@automattic/jetpack-e2e-commons",
```

Comment:
Looks like this one is going to cause similar problems to the `jetpack-js-tools` one, once the build gets fixed.